packages/gatsby-plugin-manifest/src/pluginOptionsSchema.js

```diff
@@ -246,6 +254,7 @@ export default function pluginOptionSchema({ Joi }) {
               `The label member is a string that serves as the accessible name of that screenshots object.`
             ),
           platform: platform,
+          form_factor: formFactor,
```

Comment:
I think `purpose` field would need to be repurposed as `description` for screenshots